app/services/preserved_object_handler.rb

```diff
@@ -133,6 +133,10 @@ def moab_validation_errors
       end
       handler_results.add_result(PreservedObjectHandlerResults::INVALID_MOAB, moab_error_msgs)
     end
+    if moab_errors.any?
+      reta = ReportErrorsToArgo.new("dh966tq9465")
+      reta.send_errors("moab-valid", "#{moab_errors}")
+    end
```

Comment:
I guess I envisioned the code to actually send the stuff to workflows happening in PreservedObjectHandlerResults ... but perhaps we can discuss pros and cons in person

Reply:
same here... i was thinking there'd be something similar to POHR's `#log_results`, like `#report_errors_to_workflows`, that we could call alongside `#log_errors` at the end of each of the public POH methods.